### PR TITLE
Fix send LAUNCH_APP to background app

### PR DESCRIPTION
**Pull request by AGritsevich**
After CRQ this feature was lose. Create additional sequence in case
register v4 apps to fing foregraund app and
send to it OnSystemRequest(LAUNCH_APP).
(bug-fixes) [APPLINK-11308](https://adc.luxoft.com/jira/browse/APPLINK-11308).

Please review @anosach-luxoft, @malirod, @AByzhynar , @OHerasym.

### CODE REVIEW

**Comment by AGaliuzov** [comment]
@AGritsevich please pay attention that the pull request is not buildable

File: src/components/application_manager/src/commands/hmi/sdl_activate_app_request.cc [hunk 1]
```diff
@@ -90,10 +92,13 @@ void SDLActivateAppRequest::Run() {
...
   } else {
-    const uint32_t application_id = app_id();
-    application_manager::ApplicationManagerImpl::instance()
-	->GetPolicyHandler().OnActivateApp(application_id,
-					   correlation_id());
+    if (devices_apps.first) {
+      MessageHelper::SendLaunchApp(
+          devices_apps.first->app_id(), app->SchemaUrl(), app->PackageName());
+    } else {
+      policy::PolicyHandler::instance()->OnActivateApp(application_id,
```
Comment (AByzhynar) [inline src/components/application_manager/src/commands/hmi/sdl_activate_app_request.cc]: @AGritsevich PolicyHandler is not a singltone anymore.

Comment (AGritsevich) [inline src/components/application_manager/src/commands/hmi/sdl_activate_app_request.cc]: @AByzhynar , thank you. I'll fix it soon as possible.

**Comment by AGritsevich** [comment]
Please review @anosach-luxoft, @malirod, @AByzhynar , @OHerasym.

**Comment by AByzhynar** [comment]
Reviewed

**Comment by OHerasym** [comment]
Reviewed

**Comment by AGritsevich** [comment]
Please review @anosach-luxoft or @malirod, you are last.

**Comment by anosach-luxoft** [comment]
@AGritsevich please in future do not name branches with so long names

**Comment by AGritsevich** [comment]
@anosach-luxoft, Ok.

File: src/components/application_manager/src/commands/hmi/sdl_activate_app_request.cc [hunk 2]
```diff
@@ -90,10 +92,14 @@ void SDLActivateAppRequest::Run() {
     }
     subscribe_on_event(BasicCommunication_OnAppRegistered);
   } else {
-    const uint32_t application_id = app_id();
-    application_manager::ApplicationManagerImpl::instance()
-	->GetPolicyHandler().OnActivateApp(application_id,
-					   correlation_id());
+    if (devices_apps.first) {
+      MessageHelper::SendLaunchApp(
+          devices_apps.first->app_id(), app->SchemaUrl(), app->PackageName());
```
Comment (anosach-luxoft) [inline src/components/application_manager/src/commands/hmi/sdl_activate_app_request.cc]: @AGritsevich are you sure we have to send `LAUNCH_APP` to already registered application?

**Comment by anosach-luxoft** [comment]
@dev-gh please participate

File: src/components/application_manager/src/commands/hmi/sdl_activate_app_request.cc (hunk 2 above)
Comment (AGritsevich) [inline src/components/application_manager/src/commands/hmi/sdl_activate_app_request.cc]: @anosach-luxoft, yes. I've saw SRQ for it. If you want, I can try to find it.

**Comment by dev-gh** [comment]
Reviewed

**Pull request merged**